dash/development/build_process.py

```diff
@@ -97,7 +97,8 @@ def digest(self):
             logger.info("bundles in %s %s", folder, copies)
 
             for copy in copies:
-                payload[f"MD5 ({copy})"] = compute_md5(self._concat(folder, copy))
+                # note md5 has been replaced with sha256, leaving string 'MD5 (hash)' as is because impacts are unclear
+                payload[f"MD5 ({copy})"] = compute_hash(self._concat(folder, copy)) 
```

Comment:
@T4rk1n @alexcjohnson Can this string be safely changed? By my understanding it shouldn't cause any issues, but looking for confirmation.

Reply:
It's only used for manual comparison:
> `renderer digest` computes the content hash of each asset in `dash_renderer` folder, prints out the result in logs, and dumps into a JSON file `digest.json`. Use this when you have a doubt about the current assets in `dash_renderer`, and compare it with previous result in one shot by this command.

It should contains the new algorithm.

Reply:
Thanks @T4rk1n .

@caplinje-NOAA , can you change this string to `SHA256`?

Reply:
@emilykl I corrected the payload string relative to this, let me know if any additional edits/corrections are required. Thanks.